test/bin/pyutils/build_bootc_images.py

```diff
@@ -255,9 +289,22 @@ def process_image_bootc(groupdir, bootcfile, dry_run):
     try:
         # Redirect the output to the log file
         with open(bf_logfile, 'w') as logfile:
-            bf_imgref = common.read_file(bf_path).strip()
+            # Download the bootc image builder itself in case
+            # it requires authorization for accessing the image
+            pull_args = [
+                "sudo", "podman", "pull",
+                "--authfile", PULL_SECRET, BIB_IMAGE
+            ]
+            common.run_command_in_shell(pull_args, dry_run, logfile, logfile)
```

Comment:
Do we have to do it each time a .bootc image is processed?

Reply:
I could not find a better place to perform this operation because the `process_image_bootc` and the rest of processing are running in parallel. Putting image pull before the processing will slow down the overall procedure by creating a prerequisite for running `process_image_bootc`.

Do you have a suggestion on how to improve this?